source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -34,27 +34,25 @@ If __Switch to Production__ is at the bottom of the left-hand menu, toggle it on
 To set up your profile, you need to copy credentials from your Worldpay account
 into your GOV.UK Pay account.
 
-You will need both your [Worldpay
+You must separately sign in to both your [Worldpay
 account](https://secure.worldpay.com/sso/public/auth/login.html) and your
-[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login) open
-and signed in, in different web browser tabs.
+[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login). You
+should have both accounts open, in different web browser tabs.
 
-1. Select __Profile__ in the left-hand menu in your Worldpay account.
+1. In your Worldpay browser tab, select __Profile__ in the left-hand menu. If you
+are prompted to choose a merchant code, select the appropriate one.
 
 1. In your GOV.UK Pay browser tab, go to __My services__ to select the live
 service you want to set up. Select __Settings__, then __Account credentials__,
 then __Edit credentials__.
 
-1. In your Worldpay browser tab, select __Account__ in the left-hand
-menu.
-
-1. From the __Identification__ section, copy the Worldpay __Merchant Code__ into
+1. In Worldpay, copy the Worldpay __Merchant Code__ from __Identification__ into
 the GOV.UK Pay __Merchant ID__ field.
 
-1. Select the checkbox __Enable original username__ and select __save__.
+1. In Worldpay, make sure the checkbox __Enable original username__ is toggled. Select __save__.
```

Comment:
what exactly does toggled mean in this case - that it shouldn't be be selected? (it shouldn't be, as the new username should be used)

Reply:
I miswrote this. I think the following would be better—let me know your thoughts:

> In Worldpay, tick __Enable original username__ if it is not already ticked, and then __Save Profile__.

Reply:
but we want them to use the New Username instead, so they shouldn't tick that box

Reply:
Ah of course, I'm sorry, too many things on my mind at the same time. We should delete this sentence altogether, I think. Are you happy with this line(?):

> If you are sure you need the __Original XML Username__, use that instead.

Reply:
no worries! i agree it's easier this way. 
I think if we include the line about Original XML username, we should make it clear that that you'll need to tick the box that says "Enable Original XML username" and click save. i can't really think of a reason they'd need to use it though.

Reply:
I agree: I think it's better without

Reply:
As it happens I deleted even the "If you are sure..." line as I guess you noticed but I think we're probably OK?